bin/rbenv-install

```diff
@@ -77,6 +79,15 @@ for option in "${OPTIONS[@]}"; do
     usage 0
     ;;
   "l" | "list" )
+    ruby-build --list
+    {
+      echo
+      echo "Only latest stable releases for each Ruby implementation are shown."
+      echo "Use 'rbenv install --list-all' to show all available versions."
```

Comment:
I think this should also be clear about "local" versions, similar to the usage message

Reply:
Good catch.

Reply:
Changed to "local".